src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -103,11 +104,14 @@ public static string RemoveVersions(string source)
         {
             // Remove semantic versions
             // Regex source: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
```

Comment:
Thoughts on keeping this comment next to the semanticVersionRegex.  It would be nice to explain why two version patterns are needed.

Reply:
Updated comments [here](https://github.com/dotnet/installer/pull/17233/commits/de0b3231189e096394398163e6cc6ae8e15d6c33#diff-2ed96e14569dd53766d8c841db49d63662b897532bc8e1bb9837f87e52476f12R107).